pkg/containers/docker.go

```diff
@@ -0,0 +1,45 @@
+package containers
+
+import (
+	"github.com/fsouza/go-dockerclient"
+)
+
+type DockerRuntime struct {
+	Client *docker.Client
+}
+
+func isRelated(volume string, container *docker.Container) bool {
+	for _, mount := range container.Mounts {
+		if mount.Name == volume && mount.Driver == "dvol" {
+			return true
+		}
+	}
+	return false
+}
+
+func (runtime DockerRuntime) Related(volume string) ([]string, error) {
```

Comment:
This method only returns related and _running_ containers, which is a side-effect of what we intend to do with them (restart them). I think that's OK with the current method name, although I would refer e.g. `RelatedRunning`. However I think at least a docstring mentioning this fact would be useful to future developers.